src/service/credentials.ts

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright 2024 IBM Corporation
```

Comment:
should this be 2025?

Reply:
Addressed in https://github.com/Qiskit/qiskit-code-assistant-jupyterlab/pull/44/commits/be799280745eaaa453b8e95196e09c95d8ad4603